### PR TITLE
feat(citizen-scripting-v8): add ability to change max-old-space-size

### PR DESCRIPTION
**Pull request by Nova-px**
Adds the ability to change the flag value for max-old-space-size via a variable to prevent heap out of memory crashes

### CODE REVIEW

**Comment by AvarianKnight** [comment]
Isn't this already possible by just passing the argument to FXServer, or if you're on txAdmin passing it in the additional arguments

**Comment by AvarianKnight** [comment]
Seems I was wrong, using `--inspect` and `--trace-gc` work fine but using `--max-old-space-size=8000` does not

**Review by blattersturm — CHANGES_REQUESTED** [review]

File: code/components/citizen-scripting-v8/src/V8ScriptRuntime.cpp [hunk 1]
```diff
@@ -2839,6 +2839,13 @@ void V8ScriptGlobals::Initialize()
 #endif
 
 	const char* flags = "--turbo-inline-js-wasm-calls --expose_gc --harmony-top-level-await";
+#ifdef V8_NODE
+	auto maxOldSpaceSize = console::GetDefaultContext()->GetVariableManager()->FindEntryRaw("node_maxOldSpaceSize");
```
[the remark this answers is not in the document]
Comment (Nova-px) [reply code/components/citizen-scripting-v8/src/V8ScriptRuntime.cpp:2843]: No, I thought developers who need it could add it manually in the server.cfg

**Comment by MonkeyWhisper** [comment]
We need this, thanks! 🙌

**Comment by ahcenezdh** [comment]
Can be closed since #2377 was merged, @prikolium-cfx

**Pull request closed without merging**